static/js/components/AnnotationsTable.jsx

```diff
@@ -84,10 +84,13 @@ const AnnotationsTable = ({ annotations, spectra = [] }) => {
 
   // Curate data
   const tableData = [];
+  annotations.push(...spectrumAnnotations);
```

Comment:
If `annotations` could ever possibly be `null`, we'd need to add `annotations?.push(...`